src/corefx/System.Globalization.Native/icushim.cpp

```diff
@@ -253,8 +253,9 @@ extern "C" int32_t GlobalizationNative_LoadICU()
 #define PER_FUNCTION_BLOCK(fn, lib) \
     static_assert((sizeof(#fn) + MaxICUVersionStringLength + 1) <= sizeof(symbolName), "The symbolName is too small for symbol " #fn); \
     sprintf(symbolName, #fn "%s", symbolVersion); \
+    dlerror(); \
```

Comment:
This extra call is not needed. The dlerror returns error of the most recent call to dlsym, dlopen, etc.
Looks good otherwise.

Reply:
Thanks.  Removed the extra call.

Reply:
```
The  dlerror()  function returns a human-readable, null-terminated string describing the 
most recent error
```
So if the dlsym() did not actually fail, but somehow had spurious error related to something like signal handling, this could be a stale error.  Probably still OK as this should represent incremental progress.  Extra dlerror() call would add unnecessary overhead.